packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -23,7 +23,7 @@ Customize the Okta URL domain.
 
 ## Overview
 
-You can customize your Okta organization by replacing the Okta domain name with your own domain name. This allows you to create a seamless branded experience for your users so that all URLs look like your application.
+You can customize your Okta organization by replacing the Okta domain name with your own domain name. This allows you to create a seamless branded experience for your users so that all URLs look like your app.
```

Comment:
"This" allows you to...
We try to not just say "This". How about: "Your customized domain name allows you to..."